pallets/collator-assignment/src/tests.rs

```diff
@@ -794,15 +794,15 @@ fn assign_collators_rotation_container_chains_are_shuffled() {
 
         MockData::mutate(|m| {
             // Seed chosen manually to see the case where container 1002 is given priority
-            m.random_seed = [2; 32];
```

Comment:
Why did you change the random seed? Even if you don't change it, wouldn't it result in different order in fixed code now?

Reply:
If I don't change it collators get assigned to container 1001, and this test is supposed to check the case where they are assigned to 1002.